<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSS_AppClass</class>
 <widget class="QMainWindow" name="TSS_AppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TSS_App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="filterGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1061</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Filters</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1041</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDateEdit" name="dateFromEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateToEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="tagFilterEdit">
        <property name="placeholderText">
         <string>Filter by tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ratingFilterSpin"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnApplyFilter">
        <property name="text">
         <string>Apply Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearFilter">
        <property name="text">
         <string>Clear Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1061</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToggleDarkMode">
       <property name="text">
        <string>View Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>1061</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>1061</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblPageSize">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Page size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPageSize">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>450</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnFirstPage">
       <property name="text">
        <string>&lt;&lt; First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrevPage">
       <property name="text">
        <string>&lt; Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPage">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNextPage">
       <property name="text">
        <string>Next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLastPage">
       <property name="text">
        <string>Last &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TSS_App.qrc"/>
 </resources>
 <connections/>
</ui>
